husky_components/datagrid/main.js

```diff
@@ -1007,6 +1005,10 @@
              */
             load: function(params) {
                 this.currentUrl = this.getUrl(params);
+
+                if (!params.data) params.data = {};
+                params.data['expand'] = this.getSelectedItemIds().join(',');
```

Comment:
Can't think of any better name at the moment... I'd say implementing an option is not necessary and makes things only more complicated.